tensorflow/compiler/tf2tensorrt/convert/trt_optimization_pass.h

```diff
@@ -25,6 +25,11 @@ limitations under the License.
 
 #if GOOGLE_CUDA && GOOGLE_TENSORRT
 
+#if !IS_TRT_VERSION_GE(7, 0, 0, 0)
+    #error From version 2.6, we only support NVIDIA TensorRT version 7 or newer.
+    #error Please update your environment and relaunch the compilation.
```

Comment:
Sorry to pick on this again: the PR subject says "warning" here you use error.
If we want a warning, let's change #error here to #warning

Reply:
Oh I see my bad, I didn't see notice this one. I fixed the title of the PR.